### PR TITLE
feat: Send alert to StreamLabs on 'fork' events

### LINKED ISSUE #57: Handle forks
It would be cool if we got StreamLabs alerts when someone forks a repository

### PR DESCRIPTION
**Pull request by orestes**
Closes #57

### CODE REVIEW

File: src/reactions/github/fork.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Fork {
+	constructor(streamlabs) {
+		this.streamlabs = streamlabs;
+	}
+
+	async handle(event, payload) {
```
Comment (SantiMA10) [inline src/reactions/github/fork.js:6]: Do we need the `event`? Also, I prefer this as a single object param with destructuring :D

File: src/reactions/github/fork.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Fork {
+	constructor(streamlabs) {
```
Comment (SantiMA10) [inline src/reactions/github/fork.js:2]: Can we apply destructuring here?

File: src/reactions/github/index.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const { Fork } = require('./fork');
+
+module.exports = {
+	Fork,
```
Comment (SantiMA10) [inline src/reactions/github/index.js:4]: Nice!

**Comment by SantiMA10** [comment]
@orestes did you add the `fork` event as an acceptable event for ping?

**Comment by orestes** [comment]
@SantiMA10 just did

**Pull request merged**